docs/UserGuide.md

```diff
@@ -115,17 +114,18 @@ DESCRIPTION OF PARAMETER
 
 
 #### `ROOM_NUMBER`
-DESCRIPTION OF PARAMETER
-* FORMAT AND RESTRICTIONS WITH JUSTIFICATION
-* (if applicable) For best usage, ...
-* (if applicable) Valid examples (if not clear from above)
+Room number for a room
+* Format: `XY-YYY`, where `X` is an integer from 1-9 inclusive, while `Y` is an integer from 0-9 inclusive. The hyphen is necessary.
+    * Valid examples: 01-000, 11-100, 12-345
+    * Invalid examples: 00-000, 00-100
+* Room numbers are unique within the SunRez
+* We disallow floor numbers being 00, as that is not a common practice in Singapore. However, unit numbers can be 000.
```

Comment:
Might want to consider ending sentences with full stops.

Reply:
Any reason why it differs from UserGuide? XY-YYY in UG vs XY-ABC in help message.

![image](https://user-images.githubusercontent.com/65516485/113588553-e0f44b80-9662-11eb-9037-d26b566eb834.png)

Reply:
Right I missed that out

Reply:
Fixed